src/main/cpp/TrajMath.cpp

```diff
@@ -0,0 +1,72 @@
+#include "TrajMath.h"
+
+t34::TrajMath::TrajMath
+(
+    double note_max_velocity_mps,
+    double target_height_meters,
+    double limelight_height_meters,
+    double shooter_angle,
+    double limelight_angle
+):
+    m_note_max_velocity_mps(note_max_velocity_mps),
+    m_target_height_meters(target_height_meters),
+    m_limelight_height_meters(limelight_height_meters),
+    m_shooter_angle(shooter_angle),
+    m_limelight_angle(limelight_angle),
+    m_target_tx(0.0),
+    m_target_ty(0.0)
+{}     
+
+double t34::TrajMath::GetFiringAngleDeg()
+{
+    //                   ____________________
+    //          ⎛  v² - √ v⁴ - g(gx² + 2v²y)   ⎞
+    // θ = atan ⎜ ———————————————————————————— ⎟
+    //          ⎝               gx             ⎠
+    //
+    // Source:
+    //   Solving Ballistic Trajectories <https://www.forrestthewoods.com/blog/solving_ballistic_trajectories/>
+    //   See section “Firing Angle to Hit Stationary Target.”
+
+    const auto v2 = m_note_max_velocity_mps * m_note_max_velocity_mps;
+    const auto v4 = v2 * v2;
+    const auto x  = m_target_distance_meters;
+    const auto x2 = x * x;
+    const auto y  = m_target_height_meters;
+
+    const auto gx  = g * x;
+    const auto gx2 = g * x2;
+    const auto v2y = v2 * y;
+
+    const auto numerator = v2 - sqrt(v4 - (g * (gx2 + (2*v2y))));
+    const auto denominator = gx;
+    const auto θ = atan(numerator / denominator);
+
+    return RAD_TO_DEG(θ);
+}
+
+bool t34::TrajMath::IsInRange()
+{
+    double rate = (
+        tan(m_shooter_angle) - 
+        (g * m_target_distance_meters) / (pow(m_shooter_angle, 2) * pow(m_note_max_velocity_mps, 2))
+    );
+
+    if (rate > 0)
+    {
+        return true;
+    }
+    else
+    {
+        return false;
+    }
```

Comment:
When returning a Boolean, consider returning the result of the condition rather than writing out a whole `if`-`else` block:

```suggestion
    return rate > 0;
```